server/src/controllers/apps.controller.ts

```diff
@@ -38,15 +38,15 @@ export class AppsController {
 
   @UseGuards(JwtAuthGuard)
   @Post()
-  async create(@User() user, @Body('icon') icon: string) {
+  async create(@User() user, @Body('name') name: string, @Body('icon') icon: string) {
```

Comment:
should create a DTO and add validations here, name is mandatory field

Reply:
done